src/content/structured/get-started/custom-theme.mdx

```diff
@@ -39,9 +44,148 @@ Alternatively, you can include the `ic-theme` component and set the `color` prop
 <ic-theme color="#5c0949"></ic-theme>
 ```
 
-### Components that use the theme colour
+The components that use the 'brand' colour are:
 
 - [Footer](/components/footer)
 - [Hero](/components/hero)
 - [Side navigation](/components/side-navigation)
 - [Top navigation](/components/top-navigation)
+
+### Theme details
+
+<ComponentDetails component="ic-theme" />
+
+## Dark mode
+
+Each component's variants can be displayed using dark mode colours. This creates better colour contrast with backgrounds, borders and text for when components need to be used on dark backgrounds.
+
+Each component has been given a `theme` prop, taking values of `inherit` (default), `light` and `dark`. By setting the value to either `light` or `dark` at the component level, those components will display dark mode colours, independent of any higher level themes or system settings.
+
+By leaving the theme prop as the default value (`inherit`), it will decide the colours based on either a high-level `ic-theme` component, or if one does not exist, the browser/system settings.
+
+### Dark mode example
+
+See the [Dark mode pattern](/patterns/dark-mode) for an example of how to implement dark mode into an application. When implementing dark mode into an app without an `ic-theme` wrapper, the components will default to the browser/system settings unless overridden manually. This will need to be accounted for to prevent unintended styling issues.
```

Comment:
```suggestion
When there is no `ic-theme` wrapper present in an app, ICDS components will default to the browser/system settings. This is unless they are overridden manually.  For a full example on how to implement dark mode into an application, see our [dark mode pattern](/patterns/dark-mode). 
```
"This will need to be accounted for to prevent unintended styling issues." - what do you mean by this? What unintended styling issues are you referring to?